crates/core_simd/src/masks/bitmask.rs

```diff
@@ -100,6 +100,15 @@ impl<T: Mask, const LANES: usize> BitMask<T, LANES> {
         unsafe { core::mem::transmute_copy(&self.0) }
     }
 
+    #[inline]
+    pub fn from_bitmask<U: Mask>(bitmask: U::BitMask) -> Self {
+        assert_eq!(
+            core::mem::size_of::<T::BitMask>(),
+            core::mem::size_of::<U::BitMask>()
+        );
+        unsafe { core::mem::transmute_copy(&bitmask) }
```

Comment:
Is `Mask` a sealed trait where we control all the impls?

Reply:
Yep: https://github.com/rust-lang/stdsimd/blob/68393aa594b0d3c5bfecd37d94fde5795de78771/crates/core_simd/src/masks/mod.rs#L22